tests/integration/ceph_block_create_test.go

```diff
@@ -161,7 +161,7 @@ func (s *BlockCreateSuite) statefulSetDataCleanup(namespace, poolName, storageCl
 }
 
 func (s *BlockCreateSuite) tearDownTest(claimName string, poolName string, storageClassName string, reclaimPolicy string, accessMode string) {
-	s.testClient.BlockClient.DeletePvc(claimName, storageClassName, accessMode)
+	s.testClient.BlockClient.DeletePvc(claimName, storageClassName, accessMode, "1M")
```

Comment:
I added the argument because in integration tests we delete pvc with kubectl delete -f template.yaml. And I change the basic method GetBlockPvcDef, wich creates the PVC templates, so...